plugin/core/views.py

```diff
@@ -41,6 +42,7 @@
 import tempfile
 
 MarkdownLangMap = Dict[str, Tuple[Tuple[str, ...], Tuple[str, ...]]]
+QuickPanelKind = Tuple[int, str, str]
```

Comment:
[`sublime.Kind`s](https://www.sublimetext.com/docs/api_reference.html#sublime.Kind) are not only used for the quick panel (also for completion popup and ListInputHandler).

Maybe call it "SublimeKind" instead?

Reply:
I wonder why `sublime.Kind` is documented but doesn't seem to actually exist...

Reply:
Never mind, it exists as a type, not as a value. And only in 3.8 host.